test/scripts/transforms/test_osiris_transform.py

```diff
@@ -259,7 +103,7 @@ def test_osiris_transform_spectroscopy():
 
     assert script.value == create_expected_script(
         input_runs="[1, 2, 3]",
-        calibration_run_number="0123456",
```

Comment:
These instances maintain a string structure. It is not clear whether there is a syntax error or not, or in which instance (see previous comment)